src/util/yaml.ts

```diff
@@ -22,26 +22,30 @@ export async function validateDefinitionFile(content: string) {
 
 /**
  * A reviver function to transform json keys
- * @param key 
- * @param value 
- * @returns 
+ * @param key
+ * @param value
+ * @returns
  */
 function reviver(key: unknown, value: unknown) {
```

Comment:
wdyt?
```
function reviver(key: unknown, value: unknown) {
  return convertToArray(key, value) ||
            parseMapping(key, value) ||
            parseBuild(key, value) ||
            parseArchiveArtifacts(key, value) ||
            value
}
```

Reply:
It is definitely more compact, but is it still equally clear (just to raise the doubt)? I would say yes in this case, therefore I agree